macros/src/main/scala-3/zio/sql/macros/wherelike.scala

```diff
@@ -5,6 +5,8 @@ sealed trait WhereIsSound[WhereF, GroupByF]
 object WhereIsSound {
   final case class WhereCanBeCalled[WhereF, GroupByF]() extends WhereIsSound[WhereF, GroupByF]
 
-  implicit def materializeWhereIsSound[WhereF, GroupByF]: WhereIsSound[WhereF, GroupByF] = WhereCanBeCalled[WhereF, GroupByF]()
+  implicit def materializeWhereIsSound[WhereF, GroupByF]: WhereIsSound[WhereF, GroupByF] = {
 
+    WhereCanBeCalled[WhereF, GroupByF]()
+  }
```

Comment:
I suppose this should go back to the original formatting.

```suggestion
  implicit def materializeWhereIsSound[WhereF, GroupByF]: WhereIsSound[WhereF, GroupByF] = WhereCanBeCalled[WhereF, GroupByF]()
```